README.md

```diff
@@ -483,14 +541,89 @@ that can be used to submit SQL queries. The endpoint is accessible under
 SQL statements are sent as the body of POST requests in JSON format, where the
 SQL statement is the value of the `stmt` attribute.
 
-### Read Schemas
+**QuantumLeap** offfers an API wrapping CrateDB backend so you can also perform
+multiple types of queries directly talking to QuantumLeap. The documentation of
+the API is [here](https://app.swaggerhub.com/apis/smartsdk/ngsi-tsdb/). Mind the
+versions. If you have access to your quantumleap container (e.g. it is running
+in locahost or port-forwarding to it), you can navigate its API via 
+[http://localhosts:8668/v2/ui](http://localhosts:8668/v2/ui).
+
+When to query **CrateDB** and when **QuatumLeap**?. As a rule of thumb, prefer 
+working always with **QuantumLeap** for the following reasons:
+
+- Your experience will be closer to FIWARE NGSI APIs like Orion's.
+- Your application will not be tied to CrateDB's specifics nor QuantumLeap's 
+implementation details, which could change and break your app.
+- QuantumLeap can be easily extended to other backends and your app will get 
+compatibility for free.
+- If your deployment is distributed, you won't need to expose the ports of 
+your database to the outside.
+
+If your are sure your query is not supported by **QuantumLeap**, you may have 
+to end up querying **CrateDB**, however, please open an issue in 
+[QuantumLeap's repo](https://github.com/smartsdk/ngsi-timeseries-api/issues) 
+so the team is aware.
+
+### :three: Checking Data persistence
```

Comment:
Numbers have been added to **Requests** in all other tutorials, and **Request** is always a 4th level header. For consistency please don't use `h5` for request and limit your other headers to 3 levels.